src/app/donation-start/donation-start.component.ts

```diff
@@ -719,8 +719,8 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
     }
 
     // Else settlement is via a new or saved card (including wallets / Payment Request Buttons).
-    const result = this.paymentGroup.value.useSavedCard
-        ? await this.stripeService.confirmPaymentWithSavedMethod(this.donation, this.stripeFirstSavedMethod as PaymentMethod)
+    const result = this.selectedSavedMethod
+        ? await this.stripeService.confirmPaymentWithSavedMethod(this.donation, this.selectedSavedMethod as PaymentMethod)
```

Comment:
as discussed just now, could be deleted
```suggestion
        ? await this.stripeService.confirmPaymentWithSavedMethod(this.donation, this.selectedSavedMethod)
```